react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+import logo from './logo.svg';
+import './App.css';
+import MyButton from './MyButton';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <MyButton text="I" classes="Btn-green Btn-standard" />
+        <MyButton text="know" classes="Btn-red Btn-standard" />
+        <MyButton text="React!" classes="Btn-blue Btn-standard" />
```

Comment:
Since they all take `Btn-standard`, and I suspect the component wouldn't display properly without it, why force the caller to pass it in?